upsc/datadog_checks/upsc/data/conf.yaml.example

```diff
@@ -1,16 +1,47 @@
 init_config:
 
 instances:
+
+    ## @param ups - string - required
```

Comment:
This parameter seems never used by the check.. should we remove it?

Reply:
it was a more of a way to give YAML something. We could change this to `alias` and allow the users to override the device name, which is sometimes ugly looking from the host.